src/Tizen.NUI/src/public/ViewProperty/LayoutExtraData.cs

```diff
@@ -77,6 +77,16 @@ internal sealed class LayoutExtraData : IDisposable
         /// </summary>
         public float MaximumHeight { get; set; } = int.MaxValue;
 
+        /// <summary>
+        /// Gets or sets the margin of the view.
+        /// </summary>
+        public UIExtents Margin { get; set; } = 0;
```

Comment:
```suggestion
        public UIExtents Margin { get; set; } = UIExtents.Zero;
```

Reply:
Thank you for your comment!
I have updated the code as you suggested.